src/main/scala/faces/utils/Helpers.scala

```diff
@@ -214,6 +214,54 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
 
   }
 
+  // center the face in a way that simulates a face detection result (including uncertainty);
+  // mostly suited for deep learning applications
+  def centerFaceBoxDeepLearning(rps: RenderParameter, scaling : Double = 2): RenderParameter = {
```

Comment:
This contains a lot of duplicated code shared with 
def centerFaceBox(rps: RenderParameter, scaling : Double = 1): RenderParameter = {

seems like half of it could be done in an external shared method?